hadoop-ozone/ozone-recon/src/main/java/org/apache/hadoop/ozone/recon/api/ContainerKeyService.java

```diff
@@ -58,7 +59,7 @@
    */
   @GET
   public Response getContainerIDList() {
-    List<Long> containerIDs;
+    Set<Long> containerIDs;
```

Comment:
We might also update the comment to "Set of .." not "List of Container IDs".  The same problem for other places we changed.